cookbooks/aws-parallelcluster-environment/recipes/config/update_shared_storages.rb

```diff
@@ -83,8 +83,17 @@ def get_efs(action)
         next unless not_in_shared_storages_mapping["efs"].nil? || !not_in_shared_storages_mapping["efs"].include?(storage)
         shared_dir_array.push(storage["mount_dir"])
         efs_fs_id_array.push(storage["efs_fs_id"])
-        efs_encryption_in_transit_array.push(storage["efs_encryption_in_transit"])
-        efs_iam_authorization_array.push(storage["efs_iam_authorization"])
+        # The EFS resource expects strings for these attributes, not booleans
+        if storage["efs_encryption_in_transit"]
```

Comment:
What about avoiding if..else block as follows?

```
efs_encryption_in_transit_array.push(String(storage["efs_encryption_in_transit"]))
```

Reply:
oh thanks this is great!

Reply:
Although I'm not sure that the object is guaranteed to exist and String cannot handle empty input